provisioners/roles/python27/tasks/main.yml

```diff
@@ -11,6 +11,7 @@
     - curl-devel
     - sqlite-devel
     - xz-libs
+    - git
```

Comment:
In thinking about this more, this isn't really a Python27 dependency so much as it is a CKAN dependency.  CKAN is the only one who actually depends on that library being installed, so perhaps it makes sense to move it to that role.